### PR TITLE
news

### PR DESCRIPTION
**Pull request by Wolverin-e**
### Description
Include a summary of the change and relevant motivation/context.  
List any dependencies that are required for this change.

Fixes # [ISSUE NUMBER HERE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Documentation
- Test

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have commented my code or provided relevant documentation
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/admin.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import nEws
+
+class News(admin.ModelAdmin):
```
Comment (anshulahuja98) [inline src/main/admin.py]: Name this as NewsAdmin

File: src/main/models.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.shortcuts import render
+
+class nEws(models.Model):
```
Comment (anshulahuja98) [inline src/main/models.py]: This is extremely stupid way of naming stuff,  make it News

File: src/main/models.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.shortcuts import render
+
+class nEws(models.Model):
+    order_no = models.PositiveIntegerField()
```
Comment (anshulahuja98) [inline src/main/models.py]: set a default high value, so that one doesn't have to change order of each news

File: src/main/models.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.shortcuts import render
+
+class nEws(models.Model):
+    order_no = models.PositiveIntegerField()
+    content = models.TextField(max_length=100)
+    do_show = models.BooleanField()
```
Comment (anshulahuja98) [inline src/main/models.py]: Refactor this to something like `active`

File: src/main/models.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.shortcuts import render
+
+class nEws(models.Model):
+    order_no = models.PositiveIntegerField()
+    content = models.TextField(max_length=100)
+    do_show = models.BooleanField()
+    document = models.FileField(upload_to='./staticfiles',blank = True)
```
Comment (anshulahuja98) [inline src/main/models.py]: this needs to be fixed, also remove unnecessary spaces

File: src/main/models.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.shortcuts import render
+
+class nEws(models.Model):
+    order_no = models.PositiveIntegerField()
+    content = models.TextField(max_length=100)
+    do_show = models.BooleanField()
+    document = models.FileField(upload_to='./staticfiles',blank = True)
+
+    def __str__(self):
```
Comment (anshulahuja98) [inline src/main/models.py]: That's too big to show, instead add a title field and return it here

File: src/main/admin.py [hunk 7]
```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import nEws
+
+class News(admin.ModelAdmin):
+    list_display=('order_no','content','do_show','document')
+
+admin.site.register(nEws,News)
```
Comment (anshulahuja98) [inline src/main/admin.py]: Look the rest of my code to look at the updated syntax of `registering on the admin

File: src/main/views.py [hunk 8]
```diff
@@ -15,4 +16,10 @@ def get_context_data(self, **kwargs):
         companies = os.listdir("staticfiles/img/company-logo")
         companies = ['img/company-logo/' + image for image in companies]
         context['companies'] = companies
+        context['objt']= nEws.objects.filter(do_show=True).order_by('order_no')
         return context
+
+'''def home(request):
```
Comment (anshulahuja98) [inline src/main/views.py]: Why is this required???

File: src/main/views.py [hunk 9]
```diff
@@ -15,4 +16,10 @@ def get_context_data(self, **kwargs):
         companies = os.listdir("staticfiles/img/company-logo")
         companies = ['img/company-logo/' + image for image in companies]
         context['companies'] = companies
+        context['objt']= nEws.objects.filter(do_show=True).order_by('order_no')
```
Comment (anshulahuja98) [inline src/main/views.py]: Horrible naming of variables, refactor

File: src/spc_django/settings.py [hunk 10]
```diff
@@ -148,8 +148,12 @@
...
+    os.path.join(BASE_DIR, "staticfiles"),
+]
+
 LOGIN_REDIRECT_URL = '/'
 LOGOUT_REDIRECT_URL = '/login/'
 
-MEDIA_ROOT = os.path.join(BASE_DIR, '')
-MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+MEDIA_URL = '/staticfiles/'
```
Comment (anshulahuja98) [inline src/spc_django/settings.py]: You need to fix this, take a reference of how I am saving documents in JobAdvertisement, in the JAF field

File: src/templates/main/index.html [hunk 11]
```diff
@@ -62,21 +62,15 @@
...
-                    2. Conducted Soft Skill Training session by Mr. Ranjan Kumar Majumder and Dr. Konica Paul
-                    Chakraborty.
-                    <br><br>
-                    3. Conducted New student team interviews.
-                    <br><br>
-                    4. Conducted Learning by Sharing Session to be taken by those who have been placed this year in
-                    reputed companies or selected in top Universities for higher studies.
+                    {% if objt %}
+                    {% for obj in objt %}
+                    {{ forloop.counter }}. {{ obj.content }}
```
Comment (anshulahuja98) [inline src/templates/main/index.html]: Poor indentation, use an IDE

**Comment by anshulahuja98** [comment]
Describe the things you  have done in the PR format above
Also check the failing travis build, fix all errors

**Comment by kushthedude** [comment]
Atleast use the issue template , its not a showcase which needs to be left untouched
